### ISSUE TITLE
Compound search bar in settings not working properly

### ISSUE THREAD

**Issue by hdavid16** [DanWBR/dwsim6#328]
Tried searching by name and it sometimes works, but not always, even when the component is in the list.
Search by CAS number fails.
version 7.5.5 (Windows 10)

**Comment by DanWBR** [comment]
@hdavid16 only name search is enabled now.

**Comment by hdavid16** [comment]
Don't you want to leave this open as a future feature request (search by CAS)?

**Comment by DanWBR** [comment]
@hdavid16 I'm so tired. sorry.

**Comment by hdavid16** [comment]
I bet. You seem to be working over the clock! Thanks for the great work.

**Comment by hdavid16** [comment]
As a side note, I just tested v8 and am still seeing issues searching for compounds. I'm on Windows. I successfully searched for an added Carbon Monoxide and Hydrogen. Then I searched for "carbon dioxide" and it did not find it. I manually scrolled down until I found carbon dioxide. 

I was then able to search for some alkanes without issue afterward. I notice that the search is more reliable now on v8, but there are still some occasional glitches.
